dev-libs/highway/highway-1.2.0.recipe

```diff
@@ -13,23 +13,35 @@ Why Highway?
 HOMEPAGE="https://github.com/google/highway"
 COPYRIGHT="2020 Google LLC"
 LICENSE="Apache v2"
-REVISION="2"
-SOURCE_URI="https://github.com/google/highway/archive/refs/tags/$portVersion.tar.gz"
-CHECKSUM_SHA256="e1d47ce510429fdcf31f41697ca74fb0dcd59d933196e531a86d51751a56f4cc"
+REVISION="1"
+SOURCE_URI="https://github.com/google/highway/releases/download/$portVersion/highway-$portVersion.tar.gz"
+CHECKSUM_SHA256="58e9d5d41d6573ad15245ad76aec53a69499ca7480c092d899c4424812ed906f"
 SOURCE_FILENAME="highway-$portVersion.tar.gz"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Comment:
Format: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#libversion-and-libversioncompat

Reply:
If debugInfoPackage is involved you would need $libVersion along with $libVersionCompat.